zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -286,7 +299,7 @@ <h3 class="interpreter-title">{{setting.name}}
             <td>
               <textarea ng-if="valueform.$visible" ng-model="dep.exclusions"
                         placeholder="(Optional) comma separated groupId:artifactId list"
-                        form="valueform"
+                        e-form="valueform"
```

Comment:
e-form is used only for the directive `editable-text`.
Same for the e-msd-elastic below I guess.

There is a bit of disparity between the inputs in general, it could probably be reworked separately